cmake/cpack_settings.cmake

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+
+include(InstallRequiredSystemLibraries)
+set(CPACK_PACKAGE_NAME "ccf")
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Confidential Consortium Framework")
+set(CPACK_PACKAGE_CONTACT "ccfeng@microsoft.com")
```

Comment:
This email address is not reachable from outside from Microsoft.

Reply:
Replaced with github